network-api/networkapi/wagtailpages/models.py

```diff
@@ -160,6 +161,7 @@ class MiniSiteNameSpace(ModularPage):
         'BanneredCampaignPage',
         'CampaignPage',
         'OpportunityPage',
+        'BlogListingPage',
```

Comment:
This needs to be a subpage of the Homepage type

Reply:
Just homepage, or also others?